Demo/AuthenticatorDemoUITests/AuthenticatorDemoUITests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class AuthenticatorDemoUITests: XCTestCase {
```

Comment:
Xcode made this `final`. I don't know if there's a reason. I thought I'd leave it as is in case I find out why.